p8e-migration/sql/V70__Adjust_Encryption_Key_Columns.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE p8e.affiliate
```

Comment:
probably want to leave out the schema name in case we decide to change it in the future.

Reply:
oh, oops, put that in to make the intelliJ sql editor happy, I'll remove it